src/styles.scss

```diff
@@ -0,0 +1,45 @@
+/* You can add global styles to this file, and also import other style files */
+$primary: rgb(111, 0, 225);
+
+body {
+  margin: 0;
+  font-size: 18px;
+}
+
+.container {
+  width: 80%;
+  margin: 0 auto;
+}
+
+header {
+  background: $primary;
+  padding: 1em 0;
+
+  a {
+    color: white;
+    text-decoration: none;
+  }
+
+  a.lego {
+    font-weight: bold;
+  }
+
+  nav {
+    float: right;
+    ul {
+      list-style-type: none;
+      margin: 0;
+      display: flex;
+      color: black;
+      
+      li a {
+        padding: 1em;
+        color: black;
+
+        &:hover {
+          background: darken($primary, 10%);
+        }
+      }
```

Comment:
I can't imagine that this is where the rendering is going wrong, as it did not show the words even before I set these.